index.js

```diff
@@ -87,14 +87,14 @@ function getProjectManifest(projectDirName) {
   };
 }
 
-exports.main = function main() {
-  if (!process.argv[2]) {
+exports.main = function main(dirPath) {
+  if (!dirPath) {
     console.error(`${chalk.red("Missing project dir name.")}\n`);
     console.log(USAGE_MSG);
     process.exit(1);
```

Comment:
@saintsebastian a `process.exit(...)` when using this package as a library is super annoying (as well as logging on the console), we can throw an Error instance from here and move the `process.exit(...)` and the logging in the CLI wrapper that we have in "bin/create-webextension", how that sounds to you?